src/app.ts

```diff
@@ -184,6 +209,36 @@ function getConfigSheetValue(field: { row: number; col: number }) {
   );
 }
 
+function getConfigSheetDataRange(startCell: { row: number; col: number }) {
```

Comment:
This is probably already available in helpers/sheets.ts. Could you please reuse the methods from there or move this code to that helper?

Reply:
same as with getConfigSheetValue, the general method is in sheets.ts, but specifically fetching from config is not. Leaving it here.